style.css

```diff
@@ -0,0 +1,63 @@
+body {
+    padding: 10px 15px;
+    display: flex;
+}
+
+div.input {
+    width: calc(100% - 17em);
+}
+
+div.damageTable {
+    position: fixed;
```

Comment:
Yeah, this just turned out to be easy, and I'd rather you play with it to see how it feels and what would feel better in a future commit designed to improve the CSS